Package.swift

```diff
@@ -101,7 +101,19 @@ let package = Package(
             dependencies: ["Commands"]),
         Target(
             /** Shim tool to find test names on OS X */
-            name: "swiftpm-xctest-helper"),
+            name: "swiftpm-xctest-helper",
+            dependencies: []),
```

Comment:
I like the ability to specify an empty array here to specifically block any automatic dependencies  That seems intuitive.